scheduler/test/cook/test/scheduler/scheduler.clj

```diff
@@ -1646,122 +1735,83 @@
           ;; We filter so that fenzo only matches one job, so we should only launch the one job.
           (let [num-considerable 2
                 offers [offer-1 offer-2 offer-3]]
-            (is (run-handle-resource-offers! num-considerable offers :normal))
+            (is (run-handle-resource-offers! num-considerable offers "test-pool"))
             (is (= :end-marker (async/<!! offers-chan)))
             (is (= 1 (count @launched-offer-ids-atom)))
-            (is (= 1 (count @launched-job-ids-atom)))
-            (is (= #{"job-1"} (set @launched-job-ids-atom))))))
-
-      (let [total-spent (atom 0)]
```

Comment:
Can you explain why you needed to delete this test? (The integration test is necessarily fragile, so this is good to keep around)

Reply:
the @total-spent atom is modified at both the `launch-matched-tasks!` and `launch-matches!` level. since we are now mocking out at the `launch-matched-tasks!` level it feels weird to artificially reify `launch-matched-tasks!` to double modify the @total-spent atom.

Reply:
@kathryn-zhou changed this to `reify` the `ComputeCluster` as you suggested, and resurrected the test case.

Reply:
Fixed.